Kitodo-API/src/main/java/org/kitodo/api/docket/DocketData.java

```diff
@@ -17,30 +17,82 @@
 
 public class DocketData {
 
+    /** The metadata file. */
+    private String metadataFile;
+
+    /** The docket data of the parent. */
+    private DocketData parent;
+
     /** The name of the process. */
     private String processName;
+
     /** The id of the process. */
     private String processId;
+
     /** The name of the project. */
     private String projectName;
+
     /** The name of the used ruleset. */
     private String rulesetName;
+
     /** The creation Date of the process. */
     private String creationDate;
+
     /** A comment. */
     private String comment;
+
     /** The template properties. */
     private List<Property> templateProperties;
+
     /** The workpiece properties. */
     private List<Property> workpieceProperties;
+
     /** The process properties. */
     private List<Property> processProperties;
 
+    /**
+     * Gets the metadataFile.
+     * 
+     * @return The metadataFile.
+     */
+    public String metadataFile() {
+        return metadataFile;
+    }
+
+    /**
+     * Sets the metadataFile.
+     * 
+     * @param metadataFile
+     *            The metadata file.
+     */
+    public void setMetadataFile(String metadataFile) {
```

Comment:
Should we not use URI class for the metadata file as the file must not necessary residence on disc?

Reply:
Changed